kube_controller_manager/datadog_checks/kube_controller_manager/kube_controller_manager.py

```diff
@@ -182,6 +190,10 @@ def check(self, instance):
 
         self._perform_service_check(instance)
 
+        if self._slis_available:
+            self.log.debug('Processing kube controller manager SLI metrics')
+            self.process(self.slis_scraper_config, metric_transformers=self.sli_transformers)
```

Comment:
base on my previous comment here this part can look like:

```suggestion
        if instance.get("sli_scraper_config", None):
            self.log.debug('Processing kube controller manager SLI metrics')
            self.process(instance["sli_scraper_config"], metric_transformers=self.sli_transformers)
```